src/locale/locales/it/messages.po

```diff
@@ -7482,7 +7482,7 @@ msgstr "Questo utente è incluso nell'elenco <0>{0}</0> che hai silenziato."
 
 #: src/components/NewskieDialog.tsx:65
 msgid "This user is new here. Press for more info about when they joined."
-msgstr ""
+msgstr "Questo utente è nuovo qui. "
```

Comment:
```suggestion
msgstr "Questo utente è nuovo qui. Premi per avere maggiori informazioni su quando si è iscritto."
```
It looks like the second sentence is missing from the translated string? This is what DeepL suggests